DataFormats/GEMRecHit/src/classes_def.xml

```diff
@@ -36,6 +36,16 @@
   <class name="edm::Wrapper<edm::RangeMap<CSCDetId,edm::OwnVector<GEMCSCSegment,edm::ClonePolicy<GEMCSCSegment> >,edm::ClonePolicy<GEMCSCSegment> > >" splitLevel="0"/>
   <class name="GEMCSCSegmentRef" splitLevel="0"/>
 
+  <class name="ME0Segment" splitLevel="0" ClassVersion="11">
```

Comment:
If I remember correctly, the ```splitLevel=0``` was intended to keep our memory size lower for data products which would only be used in the RECO data tier (larger split levels usually require more output buffer memory). Therefore I believe your three points are correct.